docs/DeveloperGuide.md

```diff
@@ -281,6 +281,20 @@ Aspect: How should Interview be presented?
 - Alternative 2: Interview exists as another separate list, stored under another new list, called `InterviewBook`.
   - Pros: Tests do not need to be modified, only new tests need to be added.
   - Cons: Too much duplication of code (Another copy of ApplicationBook).
+
+### Find Feature 
+
+#### Design considerations: 
+
+Aspect: What fields should the `find` command search through? 
+
+- Alternative 1 (current choice): Find command finds all applications whose company name and/or position contain any of the specified keywords.
```

Comment:
```suggestion
- Alternative 1 (current choice): `find` command finds all applications whose company name and/or position contain any of the specified keywords.
```
Maybe you can wrap the word find just for consistency? Otherwise everything lgtm! Thanks for adding this!